src/views/pages/Analytics/index.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div>
+    <!-- Stats Section -->
+    <div class="md-layout stats md-gutter">
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Milk Collected (Month)</div>
+              <div class="stat">{{collected}} L</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Milk Delivered (Month)</div>
+              <div class="stat">{{delivered}} L</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Loans Dispensed (Month)</div>
+              <div class="stat">{{disperesed}} UGX</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+    </div>
+    <!-- Graphs -->
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="productTranscations" 
+        title="Milk-Collections-and-Deliveries" 
+        xUnits="Date" 
+        yUnits="Litres"
+      />
+    </md-card>
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="moneyTranscations" 
+        title="Loan-Dispersals-and-Payments" 
+        xUnits="Date" 
+        yUnits="UGX"
```

Comment:
probably don't need to do this since we're only dealing with UGX for now but this could be dynamically chosen

Reply:
in the future that might make sense. For the sake of this demo I think we should just leave its as UGX